src/types/tournament_options.hpp

```diff
@@ -39,9 +39,9 @@ struct Sprt {
     double beta          = 0.0;
     double elo0          = 0.0;
     double elo1          = 0.0;
-    bool logistic_bounds = false;
+    std::string model    = "normalized";
```

Comment:
can you document the available models as a comment here please?

Reply:
added 👍